plugin/session_buffer.py

```diff
@@ -112,6 +116,7 @@ def __init__(self, session_view: SessionViewProtocol, buffer_id: int, uri: Docum
         self._last_semantic_region_key = 0
         self._check_did_open(view)
         self._session.register_session_buffer_async(self)
+        self._inlay_hints_phantom_set = sublime.PhantomSet(view, "lsp_inlay_hints")
```

Comment:
Not that it matter in the current code but would be better to move it before `_check_did_open` since it technically can already do something with inlay hints (even if async).